src/app/sign-in/components/form-sign-in/form-sign-in.component.ts

```diff
@@ -82,8 +82,7 @@ export class FormSignInComponent implements OnInit, AfterViewInit {
         session = session as UserSession
         platform = platform as PlatformInfo
         this.platform = platform
-
-        if (session.oauthSession) {
+        if (session.oauthSession && this.isQueryParametersEmpty(this.platform.queryParameters)) {
```

Comment:
a- seems like this function should be called `isQueryParameters*Not*Empty`

b- even when the query parameters are empty the Oauth session should be loaded to make this one possible https://trello.com/c/mwId6nLd/7036-the-oauth-session-is-forgotten